packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/TreeItem.module.scss

```diff
@@ -0,0 +1,162 @@
+@import '@deephaven/components/scss/custom.scss';
+
+.Wrapper {
+  list-style: none;
+  box-sizing: border-box;
+  padding-left: var(--spacing);
+  margin-bottom: -1px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+
+  &:first-child {
+    padding-top: $spacer-1;
+  }
+
+  &.clone {
+    // display: inline-block;
+    pointer-events: none;
+    padding: 0;
+
+    .TreeItem {
+      opacity: 1;
+      color: $foreground;
+      background-color: $primary;
+      box-shadow: $box-shadow;
+    }
+  }
+
+  &.ghost {
+    &.indicator {
+      opacity: 1;
+      position: relative;
+      z-index: 1;
+      margin-bottom: -1px;
+
+      .TreeItem {
+        position: relative;
+        padding: 0;
+        height: 8px;
+        border-color: #2389ff;
+        background-color: #56a1f8;
+
+        &:before {
+          position: absolute;
+          left: -8px;
+          top: -4px;
+          display: block;
+          content: '';
+          width: 12px;
+          height: 12px;
+          border-radius: 50%;
+          border: 1px solid #2389ff;
```

Comment:
😞 why all these hardcoded colours?

Reply:
These aren't used. They're from the example which has an option for a smaller indicator of where you're dropping instead of a ghost item